Build crc32c for ARM

I try to build crc32c for armv5te. I am getting an error:
```
error[E0658]: macro is_x86_feature_detected! is unstable
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/crc32c-0.4.0/src/lib.rs:38:8
   |
38 |     if is_x86_feature_detected!("sse4.2") {
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/27731
   = help: add #![feature(stdsimd)] to the crate attributes to enable

error: 
        is_x86_feature_detected can only be used on x86 and x86_64 targets.
        You can prevent it from being used in other architectures by
        guarding it behind a cfg(target_arch) as follows:

            #[cfg(any(target_arch = "x86", target_arch = "x86_64"))] {
                if is_x86_feature_detected(...) { ... }
            }
        
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/crc32c-0.4.0/src/lib.rs:38:8
   |
38 |     if is_x86_feature_detected!("sse4.2") {
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error[E0432]: unresolved import `std::arch::x86_64`
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/crc32c-0.4.0/src/hw.rs:5:5
  |
5 | use std::arch::x86_64 as simd;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ no `x86_64` in `arch`

error: the feature named `sse4.2` is not valid for this target
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/crc32c-0.4.0/src/hw.rs:72:18
   |
72 | #[target_feature(enable = "sse4.2")]
   |                  ^^^^^^^^^^^^^^^^^ `sse4.2` is not valid for this target

error: the feature named `sse4.2` is not valid for this target
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/crc32c-0.4.0/src/hw.rs:78:18
   |
78 | #[target_feature(enable = "sse4.2")]
   |                  ^^^^^^^^^^^^^^^^^ `sse4.2` is not valid for this target

error: aborting due to 5 previous errors

Some errors have detailed explanations: E0432, E0658.
For more information about an error, try `rustc --explain E0432`.
error: Could not compile `crc32c`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,10 @@
 language: rust
+arch:
+  - amd64
+  - arm64
 rust:
   # Test on the minimum supported version
-  - 1.33.0
+  - 1.41.0
   # Test on the other Rust channels
   - stable
   - nightly

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "crc32c"
-version = "0.4.0"
+version = "0.5.0"
 authors = ["Zack Owens"]
 license = "Apache-2.0/MIT"
 keywords = ["crc", "simd"]
@@ -17,7 +17,7 @@ exclude = [
 build = "build.rs"
 
 [dev-dependencies]
-rand = "0.4"
+rand = { version ="0.7", features=["alloc", "getrandom"] }
 criterion = "0.3"
 
 [[bench]]

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@ First, add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-crc32c = "0.4"
+crc32c = "0.5"
 ```
 
 ```rust

--- a/benches/rand.rs
+++ b/benches/rand.rs
@@ -4,14 +4,12 @@ extern crate rand;
 extern crate crc32c;
 
 use criterion::{Criterion, Benchmark, Throughput};
-use rand::{OsRng, Rng};
+use rand::{rngs::OsRng, RngCore};
 use crc32c::crc32c;
 
 fn crc32c_megabyte(c: &mut Criterion) {
     let mut bytes = [0u8; 1_000_000];
-
-    let mut r = OsRng::new().unwrap();
-    r.fill_bytes(&mut bytes);
+    OsRng.fill_bytes(&mut bytes);
 
     c.bench(
         "crc32_update_megabytes",

